### PR TITLE
grafana/ui: convert sass to emotion

### LINKED ISSUE #52155: Explore: Convert SASS styling .elapsed-time, .logs-rows, .logs-meta-item__labels, .text-link, .label-slider, .slider, .muted, .space-between
- [x] **elapsed-time** (public/sass/pages/_explore.scss L41, public/app/features/explore/Time.tsx L12) => styling n/a; has been removed
- [x] **logs-rows** (public/app/features/explore/LiveLogs.tsx L122, packages/grafana-ui/src/components/Logs/getLogRowStyles.ts L43-48) => styling n/a; has been removed
- [x] **logs-meta-item__labels** (public/app/features/explore/LogsMetaRow.tsx L110, Where is it defined?) => styling n/a; has been removed
- [x] **text-link** (public/app/features/explore/NoDataSourceCallToAction.tsx L25 + many others, defined in packages/grafana-ui/src/themes/GlobalStyles/elements.ts L143-145) => already converted to emotion
- [x] **label-slider** (public/app/features/explore/RichHistory/RichHistoryQueriesTab.tsx L41, 174, 175, 190) => transfer to own element
- [x] **slider** (public/app/features/explore/RichHistory/RichHistoryQueriesTab.tsx L58+L60+L178, RangeSlider.tsx L33, packages/grafana-ui/src/components/Slider/Slider.tsx L85, styles.ts L23-62 (?), public/app/core/components/OptionsUI/slider.tsx L96+L98) => transfer to own element
- [x] **muted** (packages/grafana-ui/src/themes/GlobalStyles/elements.ts L88-95)  => styling n/a; has been removed
- [x] **space-between** (public/app/features/explore/RichHistory/RichHistorySettingsTab.tsx L29-31, L81, L93, L104) => transfer to own element

### PR DESCRIPTION
**Pull request by L-M-K-B**
Fixes #52155

- **elapsed-time** => styling n/a; has been removed
- **logs-rows** => styling n/a; has been removed
- **logs-meta-item__labels** => styling n/a; has been removed
- **text-link** => already converted to emotion
- **label-slider** => transfer to own element
- **slider** => transfer to own element
- **muted** => styling n/a; has been removed
- **space-between** => transfer to own element

### CODE REVIEW

File: public/app/features/explore/LogsMetaRow.tsx [hunk 1]
```diff
@@ -78,7 +78,7 @@ export const LogsMetaRow: React.FC<Props> = React.memo(
           >
             <Button variant="secondary" size="sm" onClick={onEscapeNewlines}>
               <span>{forceEscape ? 'Remove escaping' : 'Escape newlines'}&nbsp;</span>
-              <Icon name="exclamation-triangle" className="muted" size="sm" />
```
Comment (Elfo404) [inline public/app/features/explore/LogsMetaRow.tsx:81]: this changes the icon's color:
Before:
Screenshot 2022-07-22 at 13 11 59

after:
Screenshot 2022-07-22 at 13 12 19

do we want this?

File: public/app/features/explore/RichHistory/RichHistorySettingsTab.tsx [hunk 2]
```diff
@@ -78,7 +78,6 @@ export function RichHistorySettingsTab(props: RichHistorySettingsProps) {
         <Field
           label="History time span"
           description={`Select the period of time for which Grafana will save your query history. Up to ${MAX_HISTORY_ITEMS} entries will be stored.`}
-          className="space-between"
```
Comment (Elfo404) [inline public/app/features/explore/RichHistory/RichHistorySettingsTab.tsx:81]: was the removal of this intentional or did you forget 
```
className={styles.spaceBetween}
```

**Review by Elfo404 — CHANGES_REQUESTED** [review]
Great stuff! 🚀 

left a couple of comments, the icon color looks fine to me even if it's changed, would double check with @radiorental and @ivanahuckova.

File: public/app/features/explore/RichHistory/RichHistorySettingsTab.tsx (hunk 2 above)
Comment (L-M-K-B) [reply public/app/features/explore/RichHistory/RichHistorySettingsTab.tsx:81]: I checked whether it's needed and it looked the same after I disabled it. That's why I decided to delete it.

Comment (Elfo404) [reply public/app/features/explore/RichHistory/RichHistorySettingsTab.tsx:81]: notice the gap between the select and the `Change the default active tab from “Query history” to “Starred"` toggle. 

after:
Screenshot 2022-07-22 at 16 58 02
before:
Screenshot 2022-07-22 at 16 58 18

looks ok to me, but it's not the same

File: public/app/features/explore/LogsMetaRow.tsx (hunk 1 above)
Comment (L-M-K-B) [reply public/app/features/explore/LogsMetaRow.tsx:81]: Thanks @staton-hyse11.
I will remove the icon.

File: public/app/features/explore/LogsMetaRow.tsx [hunk 3]
```diff
@@ -78,7 +78,6 @@ export const LogsMetaRow: React.FC<Props> = React.memo(
           >
             <Button variant="secondary" size="sm" onClick={onEscapeNewlines}>
               <span>{forceEscape ? 'Remove escaping' : 'Escape newlines'}&nbsp;</span>
```
Comment (Elfo404) [inline public/app/features/explore/LogsMetaRow.tsx:80]: maybe we don't need the whitespace if we removed the icon?

Comment (L-M-K-B) [reply public/app/features/explore/LogsMetaRow.tsx]: I removed it.

**Review by Elfo404 — APPROVED** [review]

**Pull request merged**